src/packages/BooleanQueryTokenizer/BooleanQueryTokenizer.ts

```diff
@@ -36,6 +36,9 @@ type Token = {
     type: TokenType;
     lexeme: string;
 };
+type Extract = {
+    wordsAndPhrases: string[];
```

Comment:
Why do you think we need result as object and not just array of strings?

Reply:
That's because we are planning to extend the result of this function in the future.
Besides `wordsAndPhrases` we will need to return wildcard words and phrases separately.